src/apps/Permissions/AssignPermissionPanel.js

```diff
@@ -183,11 +181,8 @@ class AssignPermissionPanel extends React.PureComponent {
 
 export default props => (
   <PermissionsConsumer>
-    {({ getAppRoles, createPermission, grantPermission }) => (
-      <AssignPermissionPanel
-        {...props}
-        {...{ getAppRoles, createPermission, grantPermission }}
-      />
+    {({ createPermission, grantPermission }) => (
+      <AssignPermissionPanel {...props} createPermission grantPermission />
```

Comment:
`createPermission` and `grantPermission` should be passed like this:

```jsx
<AssignPermissionPanel
  {...props}
  createPermission={createPermission}
  grantPermission={grantPermission}
/> 
```

Or like this:

```jsx
<AssignPermissionPanel
  {...props}
  {...{ createPermission, grantPermission }}
/> 
```

Reply:
I reopened because:

- I think `createPermission` can be removed (it doesn’t seem to be used anymore)?
- `grantPermission` still need to be passed using `grantPermission={grantPermission}` (otherwise it’s converted to `true`).